app/controllers/friendships_controller.rb

```diff
@@ -18,6 +18,14 @@ def destroy
     redirect_to root_path
   end
 
+  # def remove_friendship
+  #   @friendship = Friendship.find_by user_id: current_user.id, friend_id: params[:id]
+  #   @friendship_reverse = Friendship.find_by friend_id: current_user.id, user_id: params[:id]
+  #   @friendship&.destroy
+  #   @friendship_reverse&.destroy
+  #   redirect_to users_path
+  # end
+
   def confirm
     @friendship = Friendship.find_by(id: params[:id])
     @friendship.update_attribute(:confirmed, true)
```

Comment:
- [ ] when friendship is confirmed, an additional row with reversed user-friend pair is created, which means your table should have a reversed row created upon confirmation. e.g
  id:1  user_id: 1 friend_id: 2 ,confirmed: true
  id:2  user_id 2 friend_id 1, confirmed: true
The friendship IDs do not need to be contiguous as users may accept requests at a later time.
The goal is to allow users to access their friends by used the friend_id.